base/docs/helpdb.jl

```diff
@@ -14645,8 +14645,7 @@ doc"""
            __precompile__(isprecompilable::Bool=true)
 
 Specify whether the file calling this function is precompilable.  If
-``isprecompilable`` is ``true``, then ``__precompile__`` throws an
-exception when the file is loaded by ``using``/``import``/``require``
+``isprecompilable`` is ``true``, then ``__precompile__`` throws an exception
```

Comment:
I have no idea how that could have happened (I used the web interface, and only edited the `norm` bit, but there it is). I'll fix it.